requirements.txt

```diff
@@ -17,4 +17,5 @@ marshmallow-sqlalchemy
 marshmallow~=3.10.0
 flask-jwt-extended==3.25.1
 sphinx
-rinohtype
\ No newline at end of file
+rinohtype
```

Comment:
where are we using this dependency ?

Reply:
I don't know how it was included but it has something to do with CSS: https://pypi.org/project/rinohtype/